### PR TITLE
unlock cookie middleware

### PR DESCRIPTION
**Pull request by hashemt**
Relates #58

### CODE REVIEW

File: server/controllers/middleware/unlockCookie.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
+
+
+exports.unlockCookie = (req, res, next) => {
+  jwt.verify(req.cookies.handyman, process.env.PRIVATEKEY, (err, decoded) => {
+    if (err) {
+      res.clearCookie('handyman');
+      res.status(401).send({ error: 'Please login again' });
```
Comment (AbdallahAmmar96) [inline server/controllers/middleware/unlockCookie.js]:
```suggestion
      res.status(401).send({ error: 'unauthorized', statusCode: 401 });
```

File: server/controllers/middleware/unlockCookie.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
+
+
+exports.unlockCookie = (req, res, next) => {
+  jwt.verify(req.cookies.handyman, process.env.PRIVATEKEY, (err, decoded) => {
```
Comment (AbdallahAmmar96) [inline server/controllers/middleware/unlockCookie.js]:
```suggestion
  jwt.verify(req.cookies.handyman, process.env.PRIVATEKEY, (err, unlockedCookie) => {
```

Comment (AbdallahAmmar96) [inline server/controllers/middleware/unlockCookie.js]:
```suggestion
  jwt.verify(req.cookies.jwt, process.env.PRIVATEKEY, (err, decoded) => {
```

File: server/controllers/middleware/unlockCookie.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const jwt = require('jsonwebtoken');
+
+exports.unlockCookie = (req, res, next) => {
+  jwt.verify(req.cookies.jwt, process.env.PRIVATEKEY, (err, unlockedCookie) => {
```
Comment (mossa-Sammer) [inline server/controllers/middleware/unlockCookie.js]: there is another folder called utils , you should put your jwt verfiy logic on it

**Pull request merged**